Add carologistics release team

* Name of the new release team: carologistics
  * [clips_vendor](https://github.com/carologistics/clips_vendor)
    * [ros/rosdistro source entry](https://github.com/ros/rosdistro/blob/8384c11cfbd2455db9115cda80545ec6593887fa/rolling/distribution.yaml#L904C1-L909C23)
    * [ ] ~There is an existing release repository which should be imported:~

* Team members
  * [TarikViehmann](https://github.com/carologistics/clips_vendor/blob/29a831ca44d7fa43736dfe01a06fd9030f0f3b91/package.xml#L7)
  * [techtasie](https://github.com/carologistics/clips_vendor/blob/29a831ca44d7fa43736dfe01a06fd9030f0f3b91/package.xml#L8)
  
Carologistics is a robotics team of students and reasearch assistants competing at RoboCup since 2012 as well as doing research in the field of robotics.
With our migration to ROS 2 we would like to make our contributions readily available to the ROS community.
As a first project, we are preparing to release our CLIPS based reasoning framework suitable for knowledge-based applications.
Tim and I have experience with general packaging in fedora and I maintain ROS binaries for Fedora for many years now (https://copr.fedorainfracloud.org/coprs/tavie/ros2).

To start, we want to get familiar with the ROS workflows by releasing the clips_vendor (the knowledge engine we use to build the framework) for humble, jazzy and rolling.

we encounter some issues related to svn during the build:
```
01:17:29   from pkg_resources import load_entry_point
01:17:29 === ./clips_vendor (svn) ===
01:17:29 Could not checkout repository 'https://svn.code.sf.net/p/clipsrules/code/branches/64x': svn: E155004: Run 'svn cleanup' to remove locks (type 'svn help cleanup' for details)
01:17:29 svn: E155004: Working copy '/tmp/ws/build_isolated/clips_vendor/clips_vendor-prefix/src/clips_vendor' locked.
01:17:29 svn: E155004: '/tmp/ws/build_isolated/clips_vendor/clips_vendor-prefix/src/clips_vendor' is already locked.
01:17:29 gmake[2]: *** [CMakeFiles/clips_vendor.dir/build.make:101: clips_vendor-prefix/src/clips_vendor-stamp/clips_vendor-download] Error 1
01:17:29 gmake[1]: *** [CMakeFiles/Makefile2:137: CMakeFiles/clips_vendor.dir/all] Error 2
01:17:29 gmake: *** [Makefile:146: all] Error 2
01:17:29 ---
```

But we are not sure what the issue is. We mainly use `ament_cmake_vendor_package` and they more-or-less hand it through to cmakes `externalProject_add`.

Could this be an issue with the build farm polling the svn working directory in multiple threads and locking it?

As it seems that barely any other packages even depends on svn I could offer to mirror the svn to git and use the git mirror for the vendor.
Not sure if that is acceptable or not.

If this is the wrong place to discuss this, just let me know.

Edit: We actually just migrated to downloading the code via zip

--- a/00-members.tf
+++ b/00-members.tf
@@ -26,6 +26,7 @@ locals {
     local.borglab_team,
     local.bosch_engineering_ohw_team,
     local.caret_team,
+    local.carologistics_team,
     local.cascade_lifecycle_team,
     local.cerberus_team,
     local.cognipilot_team,

--- a/00-repositories.tf
+++ b/00-repositories.tf
@@ -26,6 +26,7 @@ locals {
     local.borglab_repositories,
     local.bosch_engineering_ohw_repositories,
     local.caret_repositories,
+    local.carologistics_repositories,
     local.cascade_lifecycle_repositories,
     local.cerberus_repositories,
     local.cognipilot_repositories,

--- a/carologistics.tf
+++ b/carologistics.tf
@@ -1,0 +1,17 @@
+locals {
+  carologistics_team = [
+    "TarikViehmann",
+    "techtasie",
+  ]
+  carologistics_repositories = [
+    "clips_vendor-release",
+  ]
+}
+
+module "carologistics_team" {
+  source       = "./modules/release_team"
+  team_name    = "carologistics"
+  members      = local.carologistics_team
+  repositories = local.carologistics_repositories
+  depends_on   = [github_membership.members, github_repository.repositories]
+}

